Python/Product/PythonTools/PythonTools/Debugger/DebugAdapter/DebugAdapterLauncher.cs

```diff
@@ -43,9 +43,13 @@ public sealed class DebugAdapterLauncher : IAdapterLauncher {
         public void Initialize(IDebugAdapterHostContext context) => _adapterHostContext = context ?? throw new ArgumentNullException(nameof(context));
 
         public ITargetHostProcess LaunchAdapter(IAdapterLaunchInfo launchInfo, ITargetHostInterop targetInterop) {
+            //if(!CheckPythonVersion(launchInfo.LaunchJson)) {
```

Comment:
Instead of changing this here, you should update DebugLaunchHelper.CreateDebugTargetInfo, because that's shared by all debuggers launch (debugpy and mixed-mode). There's already a check for < 2.6 over there, it can be changed to < 3.0.

Reply:
Cool, that is what I have been looking for!